includes/Core/Permissions/Permissions.php

```diff
@@ -260,4 +260,24 @@ private function grant_additional_caps( array $allcaps ) {
 
 		return $allcaps;
 	}
+
+	/**
+	 * Get all capabilities used in Google Site Kit.
+	 *
+	 * @since 1.x
+	 *
+	 * @return array
+	 */
+	public static function get_capabilities() {
+		return array(
+			self::AUTHENTICATE,
+			self::SETUP,
+			self::VIEW_POSTS_INSIGHTS,
+			self::VIEW_DASHBOARD,
+			self::VIEW_MODULE_DETAILS,
+			self::MANAGE_OPTIONS,
+			self::PUBLISH_POSTS,
```

Comment:
This capability is entirely removed via #2938, so it would probably be best to already remove it from this array as well.